distribution/tools/server-cli/src/test/java/org/elasticsearch/server/cli/JvmOptionsParserTests.java

```diff
@@ -44,6 +47,19 @@ public class JvmOptionsParserTests extends ESTestCase {
 
     private static final Map<String, String> TEST_SYSPROPS = Map.of("os.name", "Linux", "os.arch", "aarch64");
 
+    private static final Path ENTITLEMENTS_LIB_DIR = Path.of("lib", "entitlement-bridge");
+
+    @BeforeClass
+    public static void beforeClass() throws IOException {
+        Files.createDirectories(ENTITLEMENTS_LIB_DIR);
+        Files.createTempFile(ENTITLEMENTS_LIB_DIR, "mock-entitlements-bridge", ".jar");
```

Comment:
Why does this need the file? The directory is not enough?

Reply:
(I'm asking because I'm a bit surprised that it needs the file, but doesn't care about the file's name.)

Reply:
:D funny you asked, you probably removed that difficult phase of the project :)
The name of the jar itself is difficult to predict (with version, snapshot, etc), so the code there test we have exactly one jar in that directory, the name can be anything.

Reply:
Of course I remember that. I was just testing whether you knew. 😅